Rock, paper, scissors/script.js

```diff
@@ -1,39 +1,39 @@
 function main() {
   let playerSelection = getPlayerSelection()
   let computerSelection = getComputerSelection();
-
-  function playRound(playerSelection, computerSelection) {
-    if (playerSelection === computerSelection) {
-        return `Computer chose ${playerSelection} too. It's a tie!`;
-    } else if (playerSelection === "rock" && computerSelection === "scissors" || 
-               playerSelection === "paper" && computerSelection === "rock" ||
-               playerSelection === "scissors" && computerSelection === "paper") {
-        return `Computer chose ${computerSelection} \nYou win, ${playerSelection} beats ${computerSelection}`;
-    } else { 
-        return `Computer chose ${computerSelection} \nYou lose, ${computerSelection} beats ${playerSelection}`;
-    }
-  }
-
   const roundResult = playRound(playerSelection, computerSelection);
   console.log(roundResult);
 }
 
 function getPlayerSelection() {
-    let playerChoice;
-    while (true) {
-      playerChoice = prompt("Rock, paper, or scissors?").toLowerCase();
-      if (playerChoice === "rock" || playerChoice === "paper" || playerChoice === "scissors") {
-        return playerChoice;
-      } else {
-        alert("Wrong choice. Please enter 'rock', 'paper', or 'scissors'.");
-      }
+  let playerChoice;
+  while (true) {
+    playerChoice = prompt("Rock, paper, or scissors?").toLowerCase();
+    if (playerChoice === "rock" || playerChoice === "paper" || playerChoice === "scissors") {
+      return playerChoice;
+    } else {
+      alert("Wrong choice. Please enter 'rock', 'paper', or 'scissors'.");
     }
   }
+}
 
 function getComputerSelection() {
-    const choices = ["rock", "paper", "scissors"];
-    const randomIndex = Math.floor(Math.random() * choices.length);
-    return choices[randomIndex];
+  const choices = ["rock", "paper", "scissors"];
+  const randomIndex = Math.floor(Math.random() * choices.length);
+  return choices[randomIndex];
+}
+
+function playRound(playerSelection, computerSelection) {
+  if (playerSelection === computerSelection) {
+    return `Computer chose ${playerSelection} too. It's a tie!`;
+  } else if (
+  playerSelection === "rock" && computerSelection === "scissors" || 
+  playerSelection === "paper" && computerSelection === "rock" ||
+  playerSelection === "scissors" && computerSelection === "paper") {
+    return `Computer chose ${computerSelection} \nYou win, ${playerSelection} beats ${computerSelection}`;
```

Comment:
Обычно, если оставляешь скобку на одной строке, а содержание скобок переносишь на другую, то содержание выделяется отступом, а каждая скобка будет на отдельной строке. 
Также при переносе оператор обычно пишут в начале строки (так удобнее читать, и я такое чаще встречал), но это зависит от кодстайла.

```suggestion
  } else if (
    playerSelection === "rock" && computerSelection === "scissors"
    || playerSelection === "paper" && computerSelection === "rock"
    || playerSelection === "scissors" && computerSelection === "paper"
  ) {
    return `Computer chose ${computerSelection} \nYou win, ${playerSelection} beats ${computerSelection}`;
```